src/utils/formatDate.ts

```diff
@@ -2,19 +2,27 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/**
- * @param {string} date
- * @param {string} locales
- */
 // TODO: Add unit test when changing this code:
 /* c8 ignore next 8 */
-function formatDate (date, locales) {
-  const jsDate = new Date(date)
+
+type DateTimeFormatOptions = {
+  year: "numeric";
+  month: "long";
+  day: "numeric";
+};
```

Comment:
You don't have to define this type yourself. If you hover over the `Intl.DateTimeFormat` constructor (which is where you get the first error), you can see the type of its second argument, and then use that directly.

[Screencast from 2024-07-24 11-36-40.webm](https://github.com/user-attachments/assets/b623da33-5aee-4aaa-8b55-e71702836df0)

(Sometimes we'll be using a library that has a type definition for a function parameter, but doesn't export it, which is a bit of a pain. In that case, you could do something like `type DateTimeFormatOptions = Parameters<typeof Intl.DateTimeFormat>[1];` as a workaround.)